### ISSUE TITLE
#23 - Train LSTM model for dynamic gesture classification

### ISSUE THREAD

**Issue by nadeemtsf** [nm-devs/hand-2-cursor#23]
## Description
Train an LSTM (Long Short-Term Memory) neural network to classify dynamic signs from landmark sequences.

### Architecture
```
LSTM(64, return_sequences=True, input_shape=(30, 42))
→ Dropout(0.3)
→ LSTM(128, return_sequences=False)
→ Dropout(0.3)
→ Dense(64, relu)
→ Dense(num_dynamic_signs, softmax)
```

### Requirements
- Install TensorFlow (`pip install tensorflow>=2.13.0`)
- Load sequence data from `data/sequences/`
- Train with categorical crossentropy + Adam optimizer
- Save model to `models/saved/model_lstm.h5`

## Acceptance Criteria
- [ ] LSTM model trains on sequence data with the architecture above
- [ ] `tensorflow` added to `requirements.txt` (commented with note about Phase 6)
- [ ] Training script prints accuracy and loss curves
- [ ] Model saved to `models/saved/model_lstm.h5`
- [ ] Achieves >80% accuracy on the test set for dynamic signs

## Roadmap Reference
> **Phase 6, Step 6.2** — *"Use LSTM neural network"*

## Order Note
Position #23. **Blocked by #22** (needs temporal sequence data to train on).
